kafka/src/main/java/io/cloudevents/kafka/impl/KafkaHeaders.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2019 The CloudEvents Authors
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.cloudevents.kafka.impl;
+
+import io.cloudevents.SpecVersion;
+import org.apache.kafka.common.header.Header;
+import org.apache.kafka.common.header.Headers;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Helper class that contains Kafka headers that should be attached either a binary or structured event message.
+ *
+ * @author Francesco Guardiani
+ * @author Florian Hussonnois
+ */
+public class KafkaHeaders {
+
+    public static final String CE_PREFIX = "ce_";
+
+    public static final String CONTENT_TYPE = "content-type";
+
+    public static final Map<String, String> ATTRIBUTES_TO_HEADERS = Stream.concat(
```

Comment:
is this Map really necessary? What does it solve?

Reply:
It pre-caches the attribute to headers name, avoiding to allocate a new one every time.

Reply:
and it can be definitely protected

Reply:
Since it requires a code change every time a new SpecVersion is released, I'd suggest to not introduce this optimization yet, unless someone complains about this short-living strings allocated. Just my 2c.